mobile-app/lib/ui/views/learn/superblock/superblock_view.dart

```diff
@@ -28,6 +29,7 @@ class SuperBlockView extends StatelessWidget {
         appBar: AppBar(
           title: Text(superBlockName),
         ),
+        drawer: const DrawerWidgetView(),
```

Comment:
```suggestion
```

Drawer is not needed in SuperBlock view and it prevents the user from going back to Learn Landing View without adding the Learn Landing View again into the navigation stack.